### PR TITLE
fix for the autosutdown

### PR DESCRIPTION
**Pull request by udaydenduluri33**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/EUI-8640

Change description
Fix for the autoshutdown issue for AKS

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[ x  ] No

### CODE REVIEW

File: api/idamCheck.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  try {
+    const idamWebUrl = getConfigValue(SERVICES_IDAM_WEB);
+    const axiosInstance = http({} as unknown as Request);
+    const result = await axiosInstance.get(`${idamWebUrl}/o/.well-known/openid-configuration`);
+    if (!result) {
+      console.log('idam must be up to start');
+      process.exit(1);
+    }
+  } catch (err) {
+    console.log('idam must be up to start');
```
Comment (connorpgpmcelroy) [inline api/idamCheck.ts:16]: Same question - are the logs necessary?

File: api/idamCheck.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { getConfigValue } from './configuration';
+import { SERVICES_IDAM_WEB } from './configuration/references';
+import { http } from './lib/http';
+import { Request } from 'express';
```
Comment (connorpgpmcelroy) [inline api/idamCheck.ts:4]: import ordering is wrong here - express should be at top right?

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with comments

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

File: api/idamCheck.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/idamCheck.ts:16]: Logs are necessary for Seeing why the App is not up

File: api/idamCheck.ts (hunk 2 above)
Comment (udaydenduluri33) [reply api/idamCheck.ts]: Changed it now

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**